src/Runner.Common/JobServerQueue.cs

```diff
@@ -476,8 +471,27 @@ private async Task ProcessSummaryUploadQueueAsync(bool runOnce = false)
                         }
                         catch (Exception ex)
                         {
-                            Trace.Info("Catch exception during summary file upload to results, keep going since the process is best effort.");
-                            Trace.Error(ex);
+                            try
```

Comment:
I would suggest removing `try-catch` since none of the code should throw within here.

Reply:
if we have a bug, then we would like to make the runner crash and find the bug via e2e test and canary runs, we use `try-catch` only in places where we know it's going to throw an exception.

Reply:
Sure, removed.